hoomd/ExecutionConfiguration.h

```diff
@@ -417,7 +417,7 @@ class PYBIND11_EXPORT ExecutionConfiguration
         {                                                                             \
         hipError_t err_sync = hipPeekAtLastError();                                   \
         this->m_exec_conf->handleHIPError(err_sync, __FILE__, __LINE__);              \
-        auto gpu_map = this->m_exec_conf->getGPUIds();                                \
+        auto gpu_map = this -> m_exec_conf->getGPUIds();                              \
```

Comment:
I don't think its common practice to put spaces around the deference operator like this.

Reply:
Indeed, this change is not desired. However, this doesn't appear to be a documented behavior in the clang-format 17 release notes: https://releases.llvm.org/17.0.1/tools/clang/docs/ReleaseNotes.html?1=#clang-format . Without an option to cause clang-format to revert this change, there is little we can do. Thankfully, it only applies to 1 line and not all uses of `->` in the codebase. We will accept it for now and hope that a future clang-format release fixes the problem.